source/fuzz/transformation_add_constant_composite.cpp

```diff
@@ -72,6 +72,24 @@ bool TransformationAddConstantComposite::IsApplicable(
       }
       break;
     case SpvOpTypeStruct:
+      // We do not create constants of structs decorated with Block nor
+      // BufferBlock.  The SPIR-V spec does not explicitly disallow this, but it
+      // seems like a strange thing to do, so we disallow it to avoid triggering
+      // low priorty edge case issues related to it.
+      if (!ir_context->get_decoration_mgr()->WhileEachDecoration(
+              composite_type_instruction->result_id(), SpvDecorationBlock,
+              [](const opt::Instruction & /*unused*/) -> bool {
```

Comment:
A minor issue: `const opt::Instruction &` -> `const opt::Instruction&`.

Reply:
Also, you can adjust comments in the header file for the `IsApplicable` method.

Reply:
Weirdly clang-format doesn't like what you're proposing (I guess it introduced the space).  If I remove the "/*unused*/" comment then clang-format doesn't want the space, but with the comment it does!

Reply:
Comment in IsApplicable updated.

Reply:
In that case, I'm ok with whatever you think is best.